src/containers/treePage/index.tsx

```diff
@@ -79,65 +99,78 @@ const EntryMessage = styled(Paragraph)`
   color: ${TEXT_GREY};
 `;
 
-const dummyCare: TreeCare[] = [
-  {
-    date: 'March 28th',
-    message: 'Was cleared of waste and weeded',
-  },
-  {
-    date: 'March 27th',
-    message: 'Was cleared of waste and weeded',
-  },
-  {
-    date: 'March 26th',
-    message: 'Was cleared of waste and weeded',
-  },
-  {
-    date: 'March 25th',
-    message: 'Was cleared of waste and weeded',
-  },
-  {
-    date: 'March 24th',
-    message: 'Was cleared of waste and weeded',
-  },
-  {
-    date: 'March 23rd',
-    message: 'Was cleared of waste and weeded',
-  },
-];
-
-const dummyTree: SiteProps = {
-  siteId: 1,
-  blockId: 1,
-  lat: 2,
-  lng: 2,
-  city: 'Boston',
-  zip: '02115',
-  address: '100 Super Tree Way',
-  entries: [
-    {
-      updatedAt: 100000000,
-      status: 'good',
-      genus: 'biggus treebus',
-      species: 'larggeus',
-      commonName: 'large tree',
-      confidence: 'good',
-      diameter: 1000000,
-      circumference: 3140000,
-      coverage: 'leaves',
-      discoloring: true,
-      pooling: false,
-      light: true,
-      bicycle: true,
-      fence: 'yes',
-      treeNotes: 'I think it looks pretty good',
-      siteNotes: 'dirty',
-    },
-  ],
-};
+interface TreeProps {
+  readonly tokens: UserAuthenticationReducerState['tokens'];
+  readonly siteData: SiteReducerState['siteData'];
+  readonly stewardShip: TreeCare[];
+  readonly adoptedSites: ProtectedSitesReducerState['adoptedSites'];
+}
+
+interface TreeParams {
+  id: string;
+}
+
+const TreePage: React.FC<TreeProps> = ({ siteData, stewardShip, tokens }) => {
+  const dispatch = useDispatch();
+  const id = Number(useParams<TreeParams>().id);
+
+  const [stewardshipFormInstance] = Form.useForm();
+
+  useEffect(() => {
+    dispatch(getSiteData(id));
+    if (asyncRequestIsComplete(tokens)) {
+      dispatch(getAdoptedSites());
+    }
+  }, [dispatch, id, tokens]);
+
+  const onFinishRecordStewardship = (values: {
+    activityDate: moment.Moment;
+    stewardshipActivities: string[];
+  }) => {
+    const activities: ActivityRequest = {
+      date: values.activityDate.format('L'),
+      watered: values.stewardshipActivities.includes('Watered'),
+      mulched: values.stewardshipActivities.includes('Mulched'),
+      cleaned: values.stewardshipActivities.includes('Cleared Waste & Litter'),
+      weeded: values.stewardshipActivities.includes('Weeded'),
+    };
+    protectedApiClient
+      .recordStewardship(id, activities)
+      .then(() => {
+        message.success('Stwardship Recorded');
+        stewardshipFormInstance.resetFields();
+        dispatch(getSiteData(id));
+      })
+      .catch((err) =>
```

Comment:
❓ Does this work? I thought the catch in protectedApiClient prevented us from actually catching the error here.

Reply:
True, I can remove this, however I'm not sure what makes the most sense to handle possible errors